src/Listener.php

```diff
@@ -64,10 +64,48 @@ public function setTagNameCase($case)
      *                                 It uses array_walk() internally.
      *                                 The suggested signature for the argument is:
      *                                 function (string &$value, string $tag);
+     *
+     * @deprecated
      */
     public function setTagValueProcessor(callable $processor = null)
```

Comment:
Done. I'm using the stfu-operator before trigger-error as suggested by the symfony docs. Please adapt if you think this is more urgent.